packages/components/src/components/post-linkarea/post-linkarea.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, Element, h, Host } from '@stencil/core';
+import { version } from '@root/package.json';
+
+@Component({
+  tag: 'post-linkarea',
+  styleUrl: 'post-linkarea.scss',
+  shadow: true,
```

Comment:
You do not need to use a shadow DOM for this component as it does not render additional elements (only the host and its content) and the styles are minimal. 
```suggestion
```